janitor/helpers/rabbit_helpers.py

```diff
@@ -0,0 +1,61 @@
+import ssl
+from typing import Any, Generator, Sequence
+
+from pika import BlockingConnection, ConnectionParameters, PlainCredentials, SSLOptions
+
+from janitor.types import RabbitMQDetails
+
+
```

Comment:
If we use the lab-share-lib library this file wouldnt be needed.
If we don't use it, I would add tests for this file.